tests/Event/ReadModel/JSONLD/EventLDProjectorTest.php

```diff
@@ -194,6 +194,47 @@ public function it_handles_new_events_without_theme(): void
         $this->assertEquals($jsonLD, $body);
     }
 
+    /**
+     * @test
+     */
+    public function it_handles_new_events_with_an_incorrect_place_type(): void
+    {
+        $eventId = '1';
+        $eventCreated = new EventCreated(
```

Comment:
Shouldn't this test `EventCopied` instead? 🤔

Reply:
Indeed, I've refactored the test